common/functions/src/scalars/strings/regexp_instr.rs

```diff
@@ -69,7 +74,7 @@ impl Function for RegexpInStrFunction {
     }
 
     fn return_type(&self) -> DataTypePtr {
-        u64::to_data_type()
+        NullableType::arc(u64::to_data_type())
```

Comment:
Why do we return `NullableType` and `disable_passthrough_null`?

It makes the code more complex.

Reply:
Oh, it's same as snowflake did.

Reply:
Yes. If we enable the `passthrough_null` , we can't know original of the argument(eg. pos) is `0` or `null` in `eval` function.